stream-chat-android-ui-components/src/main/res/values/attrs_giphy_media_attachment_view.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Giphy Media Attachment View -->
+    <declare-styleable name="GiphyMediaAttachmentView">
+        <!-- Drawable of the media attachment progressbar -->
+        <attr name="streamUiGiphyMediaAttachmentProgressIcon" format="reference" />
+        <!-- Giphy icon drawable -->
+        <attr name="streamUiGiphyMediaAttachmentGiphyIcon" format="reference" />
+        <!-- Media attachment placeholder icon -->
+        <attr name="streamUiGiphyMediaAttachmentPlaceHolderIcon" format="reference" />
+        <!-- Media attachment background color -->
+        <attr name="streamUiGiphyMediaAttachmentImageBackgroundColor" format="color|reference" />
+        <!-- Sets the Giphy Attachment's height -->
+        <attr name="streamUiGiphyMediaAttachmentGiphyMaxHeight" format="dimension|reference" />
```

Comment:
Attribute `streamUiGiphyMediaAttachmentGiphyMaxHeight` is no longer used or read in `GiphyMediaAttachmentViewStyle`, should probably be removed.